test/PolyhedralGeometry/polyhedral_fan.jl

```diff
@@ -119,6 +119,36 @@
   end
 end
 
+@testset "minimal_supercone" begin
+  X = projective_space(NormalToricVariety, 3)
+  c = Oscar.minimal_supercone(X, [1, 1, 1])
+  @test rays(c) isa SubObjectIterator{RayVector{QQFieldElem}}
+  @test sort(collect(rays(c))) == sort([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
```

Comment:
```suggestion
  @test issetequal(rays(c)), [[1, 0, 0], [0, 1, 0], [0, 0, 1]])
```
and similar for the other tests below. Sorting needs resources that can be saved this way

Reply:
I don't think it works

```
julia>   X = projective_space(NormalToricVariety, 3)
Normal toric variety

julia>   c = Oscar.minimal_supercone(X, [1, 1, 1])
Polyhedral cone in ambient dimension 3

julia> rays(c) == [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
true

julia> issetequal(rays(c), [[1, 0, 0], [0, 1, 0], [0, 0, 1]])
false
```

Reply:
Hm, weird. I thought this was how @benlorenz adapted the other tests recently.

Reply:
Perhaps the difference of types explains the output?

```
julia> typeof(rays(c))
SubObjectIterator{RayVector{QQFieldElem}}

julia> typeof([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
Vector{Vector{Int64}}
```

Reply:
`issetequal` wont work if the hash-functions are not compatible which is not really feasible when comparing different types with different semantics.
This should work:
```suggestion
  @test issetequal(rays(c)), ray_vector.([[1, 0, 0], [0, 1, 0], [0, 0, 1]]))
```

Reply:
Updated the tests to use `issetequal` as @benlorenz suggested.

> which is not really feasible

It looks feasible to me:

```
julia> issetequal(Int64[1, 2], Int8[1, 2])
true

julia> issetequal(Int64[1, 2], ZZRingElem[1, 2])
true
```

It is not for this pull request, but perhaps the hash function of QQFieldElem should be altered.

```
julia> issetequal(Int64[1, 2], QQFieldElem[1, 2])
false
```